libs/blocks/tabs/tabs.js

```diff
@@ -57,7 +57,53 @@ export function getRedirectionUrl(linkedTabsList, targetId) {
   return currentUrl;
 }
 
-function changeTabs(e) {
+const loadActiveTabFromStorage = () => {
+  const activeTab = sessionStorage.getItem('active-tab');
```

Comment:
when there are multiple tab containers in the page, this can be confusing, I think the key should be contructed from the console.

I would be more cautious here, and require a name, to make it persistent.
basically GWP should author the name in the block.